start.go

```diff
@@ -29,9 +29,9 @@ type Panel interface {
 }
 
 const (
-	CONFIG_PATH    = ".config/gomu/config"
-	HISTORY_PATH   = "~/.local/share/gomu/urls"
-	MUSIC_PATH     = "~/music"
+	CONFIG_PATH  = ".config/gomu/config"
```

Comment:
Any reasoning to change the case style?

Reply:
I change it simply because there were several warning in neovim, said: should not use ALL_CAPS in Go names; use CamelCase instead.
But do you think UpperCamelCase is better for this? 
https://stackoverflow.com/questions/22688906/go-naming-conventions-for-const

Reply:
I mean lowerCamelCase could be better.

Reply:
I agree with changing the case style